docs/background/kubernetes/gardener/autoupgrades.md

```diff
@@ -15,6 +15,14 @@ During the upgrade, the API server may briefly be unavailable.
 Both minor and major [Garden Linux](garden-linux.md) release upgrades are allowed and performed automatically.
 Also, version jumps are allowed over minor release upgrades.
 
+Whenever there is a new machine image available, this is indicated in the {{gui}} --- and in the cluster row in particular.
```

Comment:
The clause after the em dash sounds odd. I'd suggest to drop it, and just replace it with a colon:

```suggestion
Whenever there is a new machine image available, this is indicated in the {{gui}}:
```